src/natcap/invest/ui_server.py

```diff
@@ -25,7 +22,7 @@
     v.pyname: _UI_META(
         run_name=k,
         human_name=v.humanname)
-    for k, v in natcap.invest.cli._MODEL_UIS.items()}
+    for k, v in cli._MODEL_UIS.items()}
```

Comment:
To be nitpicky, `key` and `value` or `val` instead of one-letter variables?

Reply:
good idea